packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -141,7 +141,44 @@ def test_logger_propagation(logger: logging.Logger) -> None:
     logger.debug(msg, "debug")
 
 
-def _log_arguments(
+class LogExceptionsKwargsDict(TypedDict, total=True):
+    logger: logging.Logger
+    level: int
+    msg_prefix: str
+    exc_info: bool
+    stack_info: bool
+
+
+@contextmanager
+def log_exceptions(
```

Comment:
Could you please add in the doctoring of a function in which cases I should be using this function.
Currently I have no idea where I could use it.

Reply:
way better, thanks.